packages/devextreme/js/__internal/ui/list/m_list.edit.decorator.context.ts

```diff
@@ -58,16 +63,22 @@ class EditDecoratorContext extends EditDecorator {
         },
       },
       _ignoreFunctionValueDeprecation: true,
-      height: function () { return this._$menuList ? getOuterHeight(this._$menuList) : 0; }.bind(this),
-      width: function () { return getOuterWidth(this._list.$element()); }.bind(this),
-      onContentReady: this._renderMenuContent.bind(this),
+      // @ts-expect-error ts-error
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+      height: (): number => (this._$menuList ? getOuterHeight(this._$menuList) : 0),
+      // @ts-expect-error ts-error
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+      width: (): number => getOuterWidth(this._list.$element()),
+      onContentReady: (e: EventInfo<dxOverlay<OverlayProperties>>): void => {
```

Comment:
```suggestion
      onContentReady: (e): void => {
```

Reply:
You don’t need to duplicate the type here if you see that TypeScript correctly infers it.